data.py

```diff
@@ -554,8 +560,11 @@ def parse_xml(self, path=r"C:/Program Files/Cisco/AMP/policy.xml"):
         try:
             root = self.get_root(path)
         except OSError as e:
-            root = self.pull_policy_from_sx()
-
+            if self.api_cred_valid == True:
+                root = self.pull_policy_from_sx()
+            else:
+                logging.debug("Unable to pull policy due to invalid Secure Endpoint API credentials.")
+                exit("Unable to pull policy due to invalid Secure Endpoint API credentials.")

```

Comment:
I like it better doing all error checks first, then deal with happy state...
```python
if self.api_cred_valid == False:
    logging.debug("Unable to pull policy due to invalid Secure Endpoint API credentials.")
    exit("Unable to pull policy due to invalid Secure Endpoint API credentials.")
root = self.pull_policy_from_sx()
```

Reply:
also should this be .debug or .warning or .error instead?

Reply:
Changed the exit criteria comments to .info and the default logging to INFO